src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -262,17 +264,9 @@ def _resampling_request_sender(self) -> Sender[ComponentMetricRequest]:
                 config=self._resampler_config,
             )
             self._resampling_actor = _ActorInfo(actor, channel)
+            self._resampling_actor.actor.start()
         return self._resampling_actor.channel.new_sender()
 
-    async def _start(self) -> None:
-        """Start the data pipeline actors."""
-        if self._data_sourcing_actor:
-            await self._data_sourcing_actor.actor.start()
-        if self._resampling_actor:
-            await self._resampling_actor.actor.start()
-        # The power distributing actor is started lazily when the first battery pool is
-        # created.
-
```

Comment:
IMHO this change should go to the second commit (fixing the start in data pipeline) not the first one (make start sync).